cmd/jet/main.go

```diff
@@ -98,8 +125,19 @@ Usage:
 
 		err = postgresgen.Generate(destDir, genData)
 
-	case strings.ToLower(mysql.Dialect.Name()), "mariadb":
-
+	case strings.ToLower(mysql.Dialect.Name()), "mysqlx", "mariadb":
+		if dsn != "" {
+			// Special case for go mysql driver. It does not understand schema,
+			// so we need to trim it before passing to generator
+			// https://github.com/go-sql-driver/mysql#dsn-data-source-name
+			idx := strings.Index(dsn, "://")
+			if idx != -1 {
+				dsn = dsn[idx+len("://"):]
+			}
```

Comment:
Maybe we could move schema trim code to GenerateDSN.

Reply:
done